### PR TITLE
[WIP] Create kind-1.25-sriov provider

### PR DESCRIPTION
**Pull request by orelmisan**
Create an SR-IOV provider, based on kind running K8s v1.25.

### CODE REVIEW

File: cluster-up/cluster/kind-1.25-sriov/certcreator/certsecret.go [hunk 1]
```diff
@@ -0,0 +1,159 @@
+package main
```
Comment (oshoval) [inline cluster-up/cluster/kind-1.25-sriov/certcreator/certsecret.go:1]: i think you can delete this folder we dont use cert creator anymore

**Comment by orelmisan** [comment]
[Change](https://github.com/kubevirt/kubevirtci/compare/9a900adf119b1c43f53477df9765aa680ec31afa..a0ace1ebf8ea4dba305252e0ba3755784fdece3b): Use mirrored node image.

**Comment by oshoval** [comment]
about the failures there
you might want to check https://github.com/kubevirt/kubevirt/issues/2566#issuecomment-702008004
not sure if relevant, but i dont think we saw this error on the previous lane

**Comment by orelmisan** [comment]
> about the failures there you might want to check [kubevirt/kubevirt#2566 (comment)](https://github.com/kubevirt/kubevirt/issues/2566#issuecomment-702008004) not sure if relevant, but i dont think we saw this error on the previous lane

Thank you for the heads-up @oshoval.

**Comment by oshoval** [comment]
> > about the failures there you might want to check [kubevirt/kubevirt#2566 (comment)](https://github.com/kubevirt/kubevirt/issues/2566#issuecomment-702008004) not sure if relevant, but i dont think we saw this error on the previous lane
> 
> Thank you for the heads-up @oshoval.

maybe worth to run 10 times to see if it even repeats

**Review by ormergi — CHANGES_REQUESTED** [review]
We don't want to maintain more than one SR-IOV provider at the moment, please rename the existing directory instead of duplicating it.

**Comment by oshoval** [comment]
> We don't want to maintain more than one SR-IOV provider at the moment, please rename the existing directory instead of duplicating it.

I don't think we should do that until we are running it officially and see it is stable
we can see it is not stable atm, and even if it was, we usually do have 2 versions for few time

**Comment by ormergi** [comment]
> I don't think we should do that until we are running it officially and see it is stable we can see it is not stable atm, and even if it was, we usually do have 2 versions for few time

I dont recall having more them one SRIOV provider, there is no need for it.

**Comment by oshoval** [comment]
> > I don't think we should do that until we are running it officially and see it is stable we can see it is not stable atm, and even if it was, we usually do have 2 versions for few time
> 
> I dont recall having more them one SRIOV provider, there is no need for it.

For every other provider, there is a grace period, i am against it
maybe even for sriov we had a grace, but it doesnt really matter, if you wish we can look on the history

File: cluster-up/cluster/kind-1.25-sriov/certcreator/certsecret.go (hunk 1 above)
Comment (orelmisan) [reply cluster-up/cluster/kind-1.25-sriov/certcreator/certsecret.go:1]: Lets get to improvements after we'll have the provider working :)

Comment (oshoval) [reply cluster-up/cluster/kind-1.25-sriov/certcreator/certsecret.go:1]: you can np, but this folder is not used so it should be an easy one

**Comment by orelmisan** [comment]
I agree with @oshoval.
After we'll get this provider to work, we need to give it some grace.
If it will replace the 1.22 provider immediately, every malfunction will block the CI.

**Comment by orelmisan** [comment]
@xpivarc and I opened an issue to kind: https://github.com/kubernetes-sigs/kind/issues/2999

**Comment by orelmisan** [comment]
/close
@oshoval is perusing a new k8s provider.

**Pull request closed without merging**